nucypher/characters/chaotic.py

```diff
@@ -44,9 +44,9 @@
 from nucypher.config.constants import MAX_UPLOAD_CONTENT_LENGTH, TEMPLATES_DIR
 from nucypher.crypto.powers import SigningPower, TransactingPower
 from nucypher.datastore.threading import ThreadedSession
+from nucypher.network.server import NonTLSHost
 
-
-class Felix(Character, NucypherTokenActor):
+class Felix(Character, NucypherTokenActor, NonTLSHost):
```

Comment:
Can `NonTLSHost` simply be instantiated inside the `__init__`?  I'm unsure I agree that this entity is accurately describe through inheritance, especially when considering the other Members being inherited.  

To be fair, with a bit of work, both `Character`  and `NucypherTokenActor` can be removed from `Felix` - then I don't mind the `NonTLSHost` parent so much.

Reply:
I put that there just to be clear that Felix was non TLS.